aws/nextflow-launch-templates.tf

```
@@ -29,3 +29,20 @@ resource "aws_launch_template" "nf_lt_bigdisk" {
   }
   update_default_version = true
 }
+
+resource "aws_launch_template" "nf_lt_auto_scaled_ebs" {
+  name = "nextflow-launchtemplate-auto-scaled-ebs"
+  tags = var.default_tags
+  # Amazon Linux 2 Kernel 5.10 AMI 2.0.20220426.0 x86_64 HVM gp2
+  image_id = "ami-0022f774911c1d690"
+  block_device_mappings {
+    device_name = "/dev/xvda"
+    ebs {
+      volume_size           = 10  # GiB
```

Do we want to specify gp3?
```suggestion
      volume_size           = 10  # GiB
      volume_type          = "gp3"
```

---

Specified gp3.